quantum/backlight/backlight_chibios.c

```diff
@@ -40,16 +40,24 @@
 #    endif
 #endif
 
-static PWMConfig pwmCFG = {0xFFFF, /* PWM clock frequency  */
-                           256,    /* PWM period (in ticks) 1S (1/10kHz=0.1mS 0.1ms*10000 ticks=1S) */
-                           NULL,   /* Breathing Callback */
-                           {       /* Default all channels to disabled - Channels will be configured durring init */
-                            {PWM_OUTPUT_DISABLED, NULL},
-                            {PWM_OUTPUT_DISABLED, NULL},
-                            {PWM_OUTPUT_DISABLED, NULL},
-                            {PWM_OUTPUT_DISABLED, NULL}},
-                           0, /* HW dependent part.*/
-                           0};
+#ifndef BACKLIGHT_PWM_COUNTER_FREQUENCY
+#    define BACKLIGHT_PWM_COUNTER_FREQUENCY 0xFFFF
+#endif
+
+#ifndef BACKLIGHT_PWM_PERIOD
+#    define BACKLIGHT_PWM_PERIOD 256
+#endif
+
+/* PWM Low Level driver specific fields */
+#ifndef pwm_lld_driver_fields
+#    define pwm_lld_driver_fields , 0, 0,
```

Comment:
Maybe we can decide to rely on the default zero initialization for static structs and just drop this platform-dependent part?

The current initializer does not match the STM32 version of `PWMConfig` anyway (there are 3 extra fields there now — `cr2`, `bdtr`, `dier`), so we already rely on `dier` being implicitly initialized to 0.